### DISCUSSION TITLE
Image CANNOT display

### DISCUSSION THREAD

**Discussion by nataliepjlin** [hexojs/hexo#4921] [Question / Answer] [answered]
Hi guys! I recently created this repo: https://github.com/nataliepjlin/heeexo

Everything works perfectly except the images. Initially I tried to resize the images, but after a few attempts (with adding some packages such as `$ npm install hexo-asset-image --save` and [this npm pack](https://www.npmjs.com/package/hexo-asset-link), no images can be displayed! with an error message `Markdown Image Path does not exists!`. Is there any ways that I can fix this? Or maybe restarting a new one will be easier? ..THANKS A LOT!!

![image](https://user-images.githubusercontent.com/100992247/160245850-6248a13d-d62a-4507-8178-9e069e290006.png)

**i've already enable `post_asset_folder`

**Comment by yoshinorin** [comment, marked as answer]
@nataliepjlin 

Hi :)

> maybe restarting a new one will be easier?

1. Do `hexo clean`
    a.  https://hexo.io/docs/commands#clean
2. Uninstall plugins which installed yourself.

---

My questions.

Q1:
Do you want to use `hexo-asset-image` ?

Q2: 
Do you want to uninstall `hexo-asset-image` ?

Q3:
When does the error message (`Markdown Image Path does not exists!`) appear?

**Reply by nataliepjlin** [reply]
> @nataliepjlin
> 
> Hi :)
> 
> > maybe restarting a new one will be easier?
> 
> 1. Do `hexo clean`
>    a.  https://hexo.io/docs/commands#clean
> 2. Uninstall plugins which installed yourself.
These work! Thank you:)

**Reply by yoshinorin** [reply]
PS:


https://github.com/hexojs/hexo/issues/4930 is perhaps the same problem.
